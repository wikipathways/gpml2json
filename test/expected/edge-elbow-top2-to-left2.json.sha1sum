65444458ff197f4bf764acdde976d893bc9942be ?-
