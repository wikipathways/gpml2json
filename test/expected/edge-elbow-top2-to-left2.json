{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["b233f","ba4b6","dc87f"],"drawAs":"rect","gpmlElementName":"Pathway","height":132.01342281879192,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":250.47894381641217,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/602d76d0736a32db64d3d68a305308220999e9e8c0ed899bfda2a73977428403"},"entityMap":{"ba4b6":{"id":"ba4b6","textContent":"aaa","wpType":"GeneProduct","x":158.05309436184854,"y":48.353065393550764,"fontSize":10,"height":53.66035742524116,"verticalAlign":"middle","width":62.42584945456363,"zIndex":32957,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"dc87f":{"id":"dc87f","textContent":"0007","wpType":"GeneProduct","x":5.75228633721072,"y":80.27065062208649,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32958,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"b233f":{"id":"b233f","drawAs":"ElbowLine","borderWidth":1,"zIndex":32956,"points":[{"isAttachedTo":"dc87f","attachmentDisplay":{"position":[0.75,0],"offset":[0,0]},"x":65.75228633721072,"y":80.27065062208649,"orientation":[0,-1]},{"isAttachedTo":"ba4b6","attachmentDisplay":{"position":[0,0.25],"offset":[0,0]},"x":158.05309436184854,"y":61.768154749861054,"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["dc87f","ba4b6"]}}}
